istio/istio.go

```diff
@@ -424,6 +446,37 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 		return &meshes.ApplyRuleResponse{
 			OperationId: arReq.OperationId,
 		}, nil
+	case installHipsterShopCommand:
+        go func() {
+            opName1 := "deploying"
+            if arReq.DeleteOp {
+                opName1 = "removing"
+            }
+            if err := iClient.executeHipsterShopInstall(ctx, arReq); err != nil {
+                iClient.eventChan <- &meshes.EventsResponse{
+                    OperationId: arReq.OperationId,
+                    EventType:   meshes.EventType_ERROR,
+                    Summary:     fmt.Sprintf("Error while %s the Hipster Shop", opName1),
+                    Details:     err.Error(),
+                }
+                return
+            }
+            opName := "deployed"
+            if arReq.DeleteOp {
+                opName = "removed"
+            }
+            iClient.eventChan <- &meshes.EventsResponse{
+                OperationId: arReq.OperationId,
+                EventType:   meshes.EventType_INFO,
+                Summary:     fmt.Sprintf("Book Info app %s successfully", opName),
```

Comment:
please change Book info app to something like google micro services demo application.